router/src/main/resources/interceptor/lib/interceptor.lua

```diff
@@ -468,8 +470,10 @@ function interceptor.handle_response_interceptor(response_handle, intercept_serv
         -- error thrown, exiting
         return
     end
+    debug_log_body(response_handle, "<< response path body << ", debug_log_enabled)
     modify_headers(response_handle, interceptor_response_body)
     modify_trailers(response_handle, interceptor_response_body)
+    debug_log_headers(response_handle, "<< response path headers << ", debug_log_enabled)
```

Comment:
We have to print headers after replacing the status code header - after line 481 right